extensions/braze/braze-currents-users-event.schema.json

```diff
@@ -27,29 +27,70 @@
           "type": "string",
           "description": "Timezone of the user."
         },
+        "braze:appGroupID": {
+          "title": "App Group ID",
+          "type": "string",
+          "description": "API ID of the app group associated with the user."
+        },
+        "braze:source": {
+          "title": "Source",
+          "type": "string",
+          "description": "The source of the attribution."
+        },
         "braze:user": {
           "title": "User",
           "type": "object",
           "properties": {
             "braze:userID": {
               "title": "User ID",
               "type": "string",
-              "description": "BSON id of the user that performed the event."
+              "description": "BSON ID of the user that performed the event."
             },
             "braze:externalUserID": {
               "title": "External User ID",
               "type": "string",
-              "description": "External user id of the user."
+              "description": "External user ID of the user."
             },
             "braze:emailAddress": {
               "title": "Email Address",
               "type": "string",
               "description": "Email address of the user."
+            },
+            "braze:language": {
+              "title": "Language",
+              "type": "string",
+              "description": "Language of the user."
+            },
+            "braze:country": {
+              "title": "Country",
+              "type": "string",
+              "description": "Country of the user."
+            },
+            "braze:gender": {
+              "title": "Gender",
+              "type": "string",
+              "description": "Gender of the user.",
+              "enum": [
+                "F",
+                "M",
+                "N",
+                "O",
+                "P"
+              ],
+              "meta:enum": {
+                "F": "Female",
+                "M": "Male",
+                "N": "Not applicable",
+                "O": "Other",
+                "P": "Prefer not to say"
+              }
+            },
+            "braze:phoneNumber": {
+              "title": "Phone Number",
+              "type": "string",
+              "description": "Phone number of the user in e164 format."
             }
-          },
-          "required": [
```

Comment:
Just want to make sure you want to remove the `required` attribute. Once removed we won't be able to add it back in the future.

Reply:
We do--it isn't required in all events this schema covers, just in the first set we implemented.